tests/post/steps/files/grafana_dashboard_uids.json

```diff
@@ -19,6 +19,7 @@
     "Node Exporter / USE Method / Cluster": "3e97d1d02672cdd0861f4c97c64f89b2",
     "Node Exporter / USE Method / Node": "fac67cfbe174d3ef53eb473d73d9212f",
     "Node Exporter / Nodes": "fa49a4706d07a042595b664c87fb33ea",
+    "Node Exporter / AIX": "7e0a61e486f727d763fb1d86fdd629c2",
```

Comment:
Hmm to me we want to remove this one MetalK8s cannot run on this kind of platform